### ISSUE TITLE
Alerts do not display closing animation or close button

### ISSUE THREAD

**Issue by ezpuzz** [poteto/ember-cli-flash#102]
Both Foundation and Bootstrap support closing/dismissing alerts with a close button.

This is more desired behavior than Ember destroying the view itself, although I agree that it needs properly destroyed at some point.  Both Foundation and Bootstrap provide events that can be listened to in order to perform this duty, or the alerts could just be destroyed on transition as they do now without much backlash I would think.

Also would allow things like the bootstrap example of buttons embedded within an alert.

**Comment by ChrizzDF** [comment]
Agree. Would be really cool to maybe check if the `flash.sticky` property is true and then have a closing action on the rendered icon/element. I think that closing/destroying the message by clicking anywhere on it isn't necessary. But I love this addon anyway and looking forward to see future changes! :)
